### PR TITLE
bpo-41726: Update the refcounts info of PyType_FromModuleAndSpec in refcounts.dat

### PR DESCRIPTION
**Pull request by shihai1991**
https://bugs.python.org/issue41726

### CODE REVIEW

**Comment by shihai1991** [comment]
cc @encukou

File: Doc/data/refcounts.dat [hunk 1]
```diff
@@ -2283,6 +2283,11 @@ PyType_CheckExact:PyObject*:o:0:
 PyType_FromSpec:PyObject*::+1:
 PyType_FromSpec:PyType_Spec*:spec::
 
+PyType_FromModuleAndSpec:PyObject*::+1:
+PyType_FromModuleAndSpec:PyObject*:module:0:
```
Comment (encukou) [inline Doc/data/refcounts.dat]: Shouldn't this be `+1`? The function does `Py_XINCREF` on the module object and stores it in `ht_module`.

Comment (shihai1991) [reply Doc/data/refcounts.dat]: Good Catch, Thanks, updated.

**Comment by shihai1991** [comment]
Thanks petr for your review and merge :)

**Pull request merged**